Block/Algolia.php

```diff
@@ -215,7 +215,7 @@ protected function getAddToCartUrl($additional = []): string
         return $this->_urlBuilder->getUrl('checkout/cart/add', $routeParams);
     }
 
-    protected function getCurrentLandingPage(): LandingPage|null|false
+    protected function getCurrentLandingPage(): \Algolia\AlgoliaSearch\Model\LandingPage|null|false
```

Comment:
Hey @cammonro , I know I approved it. I had an assumption that I would like to clear. The assumption I had was, it's required to have the full path here. Did you try importing this object and using the sort form? If it works with short form, I'd suggest to use that for consistency.

Reply:
@mrahman3177 Well the full import is definitely intentional to distinguish `\Algolia\AlgoliaSearch\Model\LandingPage` from the local `\Algolia\AlgoliaSearch\Block\LandingPage`. I could add an import but as it was only one occurrence I did the once off qualifier.

Reply:
I could make this really clear with an alias but let me know what you think.